azure-templates/deploy-job.yml

```diff
@@ -79,6 +79,18 @@ jobs:
         displayName: Use lerna to bump version
         condition: succeeded()
 
+      - script: |
+          set -ex
+          cd ./packages/blockchain-website
+          npm install
+          npm run build
+          rm -rf ../../docs/*
+          mv build/* ../../docs
+          rm -rf build node_modules
```

Comment:
why are you removing the build and node modules folder?

Reply:
this was because we didnt want node_modules to get committed (as this step takes place just before we commit the version bump.
and the reason the build folder was removed/deleted was because we moved all of its contents into the docs so the build folder was no longer needed

Reply:
surely if you add the build and node_modules to .gitIgnore file then they won't be committed?

Reply:
Added to .gitignore